x-pack/plugins/maps/public/routing/routes/list/list.js

```diff
@@ -0,0 +1,451 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import _ from 'lodash';
+import { getMapsSavedObjectLoader } from '../../../bootstrap/services/gis_map_saved_object_loader';
+import {
+  getMapsCapabilities,
+  getUiSettings,
+  getToasts,
+  getCoreChrome,
+} from '../../../kibana_services';
+import {
+  EuiTitle,
+  EuiFieldSearch,
+  EuiBasicTable,
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiLink,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+  EuiSpacer,
+  EuiOverlayMask,
+  EuiConfirmModal,
+  EuiCallOut,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { addHelpMenuToAppChrome } from '../../../help_menu_util';
+import { Link, withRouter } from 'react-router-dom';
+import { updateBreadcrumbs } from '../../page_elements/breadcrumbs';
+import { goToSpecifiedPath } from '../../maps_router';
+
+export const EMPTY_FILTER = '';
+
+export const MapsListView = withRouter(
```

Comment:
Why are you using `withRouter` here? This component does not use anything `withRouter` provides like `match`, 'location', or 'history'

Reply:
Good catch. It used to but no longer does. Will remove!